api/src/main/java/org/pmiops/workbench/actionaudit/auditors/CommonAuditEventInfo.kt

```diff
@@ -1,8 +0,0 @@
-package org.pmiops.workbench.actionaudit.auditors
-
-data class CommonAuditEventInfo(
-    val actionId: String,
-    val userId: Long,
-    val userEmail: String?,
```

Comment:
we only use this in one place where it is not null, so don't worry about this one